lib/matplotlib/lines.py

```diff
@@ -66,7 +66,7 @@ def _get_dash_pattern(style):
 
 
 def _scale_dashes(offset, dashes, lw):
-    if rcParams['_internal.classic_mode']:
+    if not rcParams['lines.scale_dashes']:
```

Comment:
no, this is addressing #6998 which I discovered we should have while writing up these docs.